gpflow_logo.svg

```diff
@@ -0,0 +1 @@
,0,0,1,43.51,81.37a33.38,33.38,0,0,1-25-11.13A32.82,32.82,0,0,1,10,44.72c.9-9.25,5.55-18.11,13.09-25A38.22,38.22,0,0,1,50.77,9.46,34.08,34.08,0,0,1,74.48,20.82l5.74-6.39C73.05,6.88,60.54.93,45.94.93h-.57a31,31,0,0,0-21.58,9.14A16.33,16.33,0,0,0,21,13.88l-1.44,2.86a1.9,1.9,0,0,1-.82.84l-2.81,1.5a16.17,16.17,0,0,0-4.58,3.84l0,0C4.81,31,1,40.35,1,50.2,1,71.12,18.08,90.06,44.7,90.06c8.85,0,15-2.08,18.81-4.12h0a15.56,15.56,0,0,0,6.68-7.5l1.09-2.84a2,2,0,0,1,1-1.09l2.76-1.24a16,16,0,0,0,4.23-3.09l0,0a8.48,8.48,0,0,0,2.46-6v-26Z"/><path class="cls-1" d="M66,30.27l6.84-7.61-.1-.09A31.69,31.69,0,0,0,50.64,11.93a35.74,35.74,0,0,0-25.91,9.65C17.65,28,13.29,36.33,12.45,45a30.37,30.37,0,0,0,7.92,23.62C32.65,82.09,53,82.33,67.73,69.14l.18-.17a7.48,7.48,0,0,0,2.39-5.51V48.9H40.37v7.86H62l1.36,9.05c-5.07,4.46-12.63,4.44-18.07,4.44-15.1,0-23.89-11.75-23.89-24.38,0-6.91,1.41-14.89,6.11-19.5,4-3.9,10.64-5,17.16-5A29.25,29.25,0,0,1,66,30.27Z"/><path class="cls-1" d="M133.21,2.17h-19.6a7.67,7.67,0,0,0-6.94,4.38h0l-1.19,2.79a2.09,2.09,0,0,1-1,1.08l-2.73,1.28A15.55,15.55,0,0,0,94.63,19v0a12.08,12.08,0,0,0-1,5.22V88.82h9.28V25.44c0-6.55,4-11.09,6.35-13.22a7.87,7.87,0,0,1,5.29-2h15.39c12.81,0,21.41,7.49,21.41,18.64,0,8.72-9.09,17.24-14.5,21.55a11.73,11.73,0,0,1-7.24,2.52H114.64V50.45h14.94a9.2,9.2,0,0,0,5.7-2c4.08-3.24,13.57-11.64,13.57-19.61,0-9.82-7.44-16.17-18.94-16.17H114.52a5.43,5.43,0,0,0-3.64,1.37c-2.06,1.85-5.52,5.79-5.52,11.38V88.82h9.28V61h16.82a15.32,15.32,0,0,0,10.81-4.47,16.54,16.54,0,0,0,3.06-4.28l1.36-2.89a2,2,0,0,1,.8-.86l2.76-1.57A15.86,15.86,0,0,0,155.44,42h0a27.61,27.61,0,0,0,4.75-15.75C160.2,13.19,149.67,2.17,133.21,2.17ZM114.64,20.8v-.06h.1c.84-.68,3,0,4.85,0h8.67c7.3,0,11.51,5,11.51,10.28,0,2.78.35,7.16-1.8,9.38-1.95,2-6.23,2-9.71,2H114.64V25.69C114.64,23.78,113.93,21.55,114.64,20.8Z"/><polygon class="cls-1" points="389.89 25.57 373.44 81.09 370.22 81.09 355.07 29.92 339.91 81.09 336.69 81.09 320.24 25.57 311.13 25.57 329.76 88.83 344.05 88.83 355.07 52.64 366.08 88.83 380.37 88.83 399 25.57 389.89 25.57"/><polygon class="cls-1" points="371.83 77.81 387.31 25.57 378.2 25.57 371.5 54.01 363.31 25.57 363.3 25.57 356.36 25.57 371.83 77.81"/><polygon class="cls-1" points="338.3 77.81 353.77 25.57 346.83 25.57 346.82 25.57 338.63 54.01 331.93 25.57 322.82 25.57 338.3 77.81"/><path class="cls-2" d="M145.9,41.94a4.32,4.32,0,1,0,.9,3.19A4.34,4.34,0,0,0,145.9,41.94Z"/><path class="cls-2" d="M26.79,19.13a4.32,4.32,0,1,0,.9,3.19A4.33,4.33,0,0,0,26.79,19.13Z"/><path class="cls-2" d="M112.59,11.93a4.33,4.33,0,1,0-3.4,7,4.33,4.33,0,0,0,4.29-3.81A4.33,4.33,0,0,0,112.59,11.93Z"/><path class="cls-2" d="M71,67.72a4.32,4.32,0,1,0,.9,3.19A4.34,4.34,0,0,0,71,67.72Z"/><path class="cls-2" d="M192.77,11.25a4.33,4.33,0,1,0-3.4,7,4.33,4.33,0,0,0,4.29-3.81A4.33,4.33,0,0,0,192.77,11.25Z"/><path class="cls-2" d="M295.34,71.16a4.32,4.32,0,1,0,.9,3.19A4.35,4.35,0,0,0,295.34,71.16Z"/><path class="cls-2" d="M265.11,37.91a4.31,4.31,0,1,0,.9,3.2A4.35,4.35,0,0,0,265.11,37.91Z"/></svg>
```

Comment:
could you move these image files to the `doc/source/_static`

Reply:
done